### PR TITLE
Feature/Add dialog when clicking on the title

### PR DESCRIPTION
**Pull request by saadii00**

### CODE REVIEW

File: apps/web/app/explore/page.tsx [hunk 1]
```diff
@@ -27,9 +33,14 @@ async function Explore() {
               className="m-2 p-4 bg-neutral-700 text-neutral-300 rounded-lg space-y-4"
```
Comment (anisharaz) [inline apps/web/app/explore/page.tsx:33]: Wrap this whole div inside the trigger.

File: apps/web/package.json
Comment (anisharaz) [inline apps/web/package.json:1]: `package.json` file is changed but the lock file is not changed , you had probability run the git commit from inside the `/apps/web/` directory. please fix this inconsistency.

**Review by anisharaz — CHANGES_REQUESTED** [review]

File: apps/web/package.json
Comment (saadii00) [reply apps/web/package.json:1]: I have not added the lock file by myself as it is automatically created by the build command using package.json

Comment (saadii00) [reply apps/web/package.json:1]: If you'd like me to add the lock file as well i would be happy to do that.

Comment (anisharaz) [reply apps/web/package.json:1]: please add it , since it helps in package version consistency.

File: apps/web/app/explore/page.tsx (hunk 1 above)
Comment (saadii00) [reply apps/web/app/explore/page.tsx]: Done.

File: apps/web/package.json
Comment (saadii00) [reply apps/web/package.json:1]: Done.

File: pnpm-lock.yaml
Comment (anisharaz) [inline pnpm-lock.yaml:1]: Although there are no removals of package although the file has changed a lot , a lot of addition and removal are made in the lock file. There is something wrong with it , please fix this.

**Review by anisharaz — CHANGES_REQUESTED** [review]

**Comment by anisharaz** [comment]
It is resolved now

**Pull request closed without merging**